### PR TITLE
Enable SiP

### PR DESCRIPTION
**Pull request by cehsu**

### CODE REVIEW

File: test/edu-benefits/components/IntroductionPage.unit.spec.jsx [hunk 1]
```diff
@@ -1,34 +0,0 @@
-import React from 'react';
-import { findDOMNode } from 'react-dom';
-import { expect } from 'chai';
-import sinon from 'sinon';
-import ReactTestUtils from 'react-dom/test-utils';
-import { IntroductionPage } from '../../../src/js/edu-benefits/1995/components/IntroductionPage';
-
-describe('Edu 1995 <IntroductionPage>', () => {
```
Comment (cehsu) [inline test/edu-benefits/components/IntroductionPage.unit.spec.jsx]: Only the 1995 had this test, which was the only test in it's directory. The e2e tests seem to supersede this, please let me know if it makes sense to revert this removal.

Comment (jbalboni) [reply test/edu-benefits/components/IntroductionPage.unit.spec.jsx]: I think that's fine, that test is outdated. I'm going to create a tech debt ticket to write new unit tests for the intro pages, though. We should have some.

File: src/js/edu-benefits/1995/components/IntroductionPage.jsx [hunk 2]
```diff
@@ -69,28 +68,31 @@ class IntroductionPage extends React.Component {
...
-              afterText="»"/>
-          </div>
-        </div>
-        <div className="omb-info--container">
+        <SaveInProgressIntro
+          messages={this.props.route.formConfig.savedFormMessages}
+          pageList={this.props.route.pageList}
+          {...this.props.saveInProgressActions}
+          {...this.props.saveInProgress}>
+          Please complete the 22-1990 form to apply for education benefits.
```
Comment (jbalboni) [inline src/js/edu-benefits/1995/components/IntroductionPage.jsx]: I think this should be "Please complete the 22-1995 form to apply to manage your education benefits." based on the 5495 form language that Peggy gave us.

**Review by jbalboni — APPROVED** [review]
One minor issue, but LGTM after that.

File: src/js/edu-benefits/1995/components/IntroductionPage.jsx (hunk 2 above)
Comment (cehsu) [reply src/js/edu-benefits/1995/components/IntroductionPage.jsx]: Thanks @jbalboni, changed.

File: test/edu-benefits/components/IntroductionPage.unit.spec.jsx (hunk 1 above)
Comment (cehsu) [reply test/edu-benefits/components/IntroductionPage.unit.spec.jsx]: Thanks for the recommendation, I've created the issue [here](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/4864).

**Pull request merged**